crates/polkavm-test-data/build.rs

```diff
@@ -0,0 +1,64 @@
+use std::path::Path;
+use std::process::Command;
+
+fn build(project: &str, profile: &str, target: &str) -> Result<(), String> {
+    let polkavm = Path::new(std::env!("CARGO_MANIFEST_DIR")).join("../..");
+    let project_path = polkavm.join("guest-programs").join(project).into_os_string().into_string().unwrap();
+
+    let remap = std::format!(
+        "--remap-path-prefix={}= --remap-path-prefix={}=",
+        polkavm.into_os_string().into_string().unwrap(),
+        std::env!("HOME"),
+    );
+
+    let mut cmd = Command::new("cargo");
```

Comment:
Not sure if it resolves the windows problem but it seems best practice to do this in build scripts:

```suggestion
    let mut cmd = Command::new(std::env!("CARGO"));
```

Reply:
thanks, agreed